src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.setColor(color);
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The `draw()` method should provide more details about the figure being drawn, such as its color and size, to match the output of `getFigureData()`.